go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.0
 	github.com/hashicorp/terraform v0.12.5
 	github.com/kr/pretty v0.1.0
-	github.com/zorkian/go-datadog-api v2.25.0+incompatible
+	github.com/zorkian/go-datadog-api v2.26.0+incompatible
```

Comment:
Please rebase to latest master which already has this version.

Reply:
can do! I'll take care of this soon.